liberty-maven-plugin/pom.xml

```diff
@@ -243,8 +227,6 @@
                             <artifactId>maven-invoker-plugin</artifactId>
                             <configuration>
                                 <pomExcludes>
-            <!-- TEMPORARILY EXCLUDE dev-it for all platforms until issue 1048 is resolved. -->
```

Comment:
It is also excluded further down in a Windows platform section. Can that be reenabled now too?

Reply:
The dev-it test did not run for the Windows platform due to the section I referenced above. Please add it and see if they pass.

Reply:
Yes, I'll do that.